README.md

```diff
@@ -0,0 +1,25 @@
+# Welcome
+
+This repository is the home for the content found at https://developers.eos.io/welcome
```

Comment:
Would be nice to add some info like:

1. This repo uses the Docs-as-Code methodology (some background info)
2. How to contribute (with some screenshots)
3. Build status of CI/CD?

Or maybe we can merge this for now and I can open a separate PR with the above items.

Reply:
we already have a CONTRIBUTING.md file in the repo.
build status of CI/CD ... I do not follow this can you elaborate?
as for one yeah we could write 4 pages about this repo but it is not in the scope of this PR.

Reply:
<img width="556" alt="Screenshot 2020-07-30 at 10 23 37 PM" src="https://user-images.githubusercontent.com/36563110/88934595-65f8bb00-d2b3-11ea-8b79-3dc37282ee0a.png">


Build status showing the status of your buildkite pipleline something like in the screenshot. But it was just something extra and as you mentioned the scope of this PR is minimal.